app/components/assignment/rows/row.js

```diff
@@ -46,15 +46,17 @@ const styles = StyleSheet.create({
   row: {
     padding: 20,
     borderBottomColor: Colors.Gray.Light,
-    borderBottomWidth: 1
+    borderBottomWidth: 1,
   },
   inset: {
     flexDirection: 'row',
     alignItems: 'center'
   },
   content: {
     justifyContent: 'space-between',
-    marginLeft: 20
+    marginLeft: 20,
+    flex: 1,
+    textOverflow: '...'
```

Comment:
Are you sure this does anything? I don't think this is an actual style rule -- i think `flex: 1` is all that's needed